src/ffi/object.rs

```diff
@@ -1,30 +1,18 @@
+// FFI note: this file changed a lot between 3.6 and 3.10.
+// Some missing definitions may not be marked "skipped".
```

Comment:
👍 yep good idea; over time I'm sure that we'll eventually fill in any gaps users need!